contrib/verify-commits/README.md

```diff
@@ -24,3 +24,24 @@ keys:
 Note that the above isn't a good UI/UX yet, and needs significant improvements
 to make it more convenient and reduce the chance of errors; pull-reqs
 improving this process would be much appreciated.
+
+Configuration files
```

Comment:
These files aren't up to date already afaik, so yeah, this could be done in another PR. Just noticing that we should probably ensure consistency between docs and actual files at some point.